perfmetrics/scripts/ls_metrics/listing_benchmark.py

```diff
@@ -35,7 +35,7 @@
 import time
```

Comment:
Please also run test files for python scripts/modules under perfmetrics/scripts and add in the description of this PR.

Reply:
reverted the changes